src/picker.js

```diff
@@ -333,65 +348,65 @@ function createImageCheckbox(url, frameId, tabId, referrer, alt) {
 	}
 	
 	async function load() {
-    let origin;
```

Comment:
It seems that your editor converts all spaces into tabs. Try to setup some plugins/scripts so it can obey [`.editorconfig`](https://editorconfig.org/).

Reply:
I have extension "EditorConfig for VS Code" enabled. Maybe it got messed up due to some extensions or settings conflict.